local-cli/core/config/windows/generateGUID.js

```diff
@@ -0,0 +1,10 @@
+const s4 = () => {
+  return Math.floor((1 + Math.random()) * 0x10000)
+    .toString(16)
+    .substring(1);
+}
+
+module.exports = function generateGUID() {
```

Comment:
>generateGUID [](start = 26, length = 12)

Nice, I should use this and remove the dependency on `uuid` in react-native-windows CLI.